Graphical Stack Survey 20231031

RedL0tus opened the pull request:
<!-- For description on topic creation and maintenance, please refer to [this Wiki article](https://wiki.aosc.io/developer/packaging/topic-based-maintenance-guideline/). -->

Topic Description
-----------------

Graphical Stack Survey 20231031
<!-- Please input topic description here. -->

Package(s) Affected
-------------------

xcb-proto: update to 1.16.0
x11-proto: update to 2023.2
libxau: update to 1.0.11
libxdmcp: update to 1.1.4
libxcb: update to 1.16
xcb-util: update to 0.4.1
xcb-util-cursor: update to 0.1.4
xcb-util-image: update to 0.4.1
xcb-util-keysyms: update to 0.4.1
xcb-util-renderutil: update to 0.3.10
xcb-util-wm: update to 0.4.2
x11-lib: update to 7.7.2020231031
libdrm: update to 2.4.117
libvdpau: update to 1.5
libva: update to 2.20.0
wayland-protocols: update to 1.32
wayland: update to 1.22.0
libva-utils: update to 2.20.0
libva-nvidia-driver: update to 0.0.10
libglvnd: update to 1.7.0
s2tc: update to 1.0+git20210317
spirv-headers: update to 1.3.268.0
spirv-tools: update to 2023.5~rc1
libclc: update to 17.0.3
glslang: update to 13.1.1
util-macros: update to 1.20.0
x11-app: update to 7.7.20231031
xbitmaps: update to 1.1.3
xkeyboard-config: update to 2.40
libxkbcommon: update to 1.6.0
pixman: update to 0.42.2
libepoxy: update to 1.5.10
x11-font: update to 7.7.20231106
libxcvt: update to 0.1.2
xorg-server: update to 21.1.9
libinput: update to 1.24.0
xf86-input-libinput: update to 1.4.0
xf86-input-vmmouse: update to 13.2.0
xf86-input-wacom: update to 13.4.0
xf86-video-amdgpu: update to 23.0.0
xf86-video-ati: update to 22.0.0
xf86-video-dummy: update to 0.4.1
xf86-video-qxl: update to 0.1.6
xf86-video-r128: update to 6.12.1
xf86-video-vesa: update to 2.6.0
xf86-video-vmware: update to 13.4.0
vulkan-headers: update to 1.3.269
vulkan-loader: update to 1.3.269
vulkan-utility-libraries: new, 1.3.269
vulkan-validationlayers: update to 1.3.269
vulkan: bump REL
glu: update to 9.0.3
glfw: update to 3.3.8
freeglut: update to 3.4.0
egl-wayland: update to 1.1.13
xwayland: update to 23.2.2
libdisplay-info: new, 0.1.1
aml: new, 0.3.0
neatvnc: new, 0.7.1
weston: update to 12.0.2
vkd3d: update to 1.9
tinyx: update to 1.3.0+git20221106
xcursor-themes: update to 1.0.7
xinit: update to 1.4.2
xsettingsd: update to 1.0.2
peg: new, 0.1.18
intel-gpu-tools: update to 1.28
glmark2: update to 2023.01
mesa-demos: update to 9.0.0
<!-- Please list all package(s) affected by this topic here. -->

Security Update?
----------------

<!-- If this topic contains security update(s), please uncomment "Yes,"
     mark with the `security` and `priority` labels, and make sure to mark your commits to relevant issue numbers.

     Please see GitHub's documentation on "Linking a pull request to an issue":

     https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue -->

<!-- Yes - Issue Number: ISSUENUMBER -->
No

<!-- Please uncomment the "Build Order" section if your topic affects more than one package. -->

Build Order
-----------

<!-- Please describe in what order maintainers should build this pull request. You can use the following template, use space to separate packages. -->

```
xcb-proto x11-proto libxau libxdmcp libxcb xcb-util xcb-util-cursor xcb-util-image xcb-util-keysyms xcb-util-renderutil xcb-util-wm x11-lib libdrm libvdpau libva wayland-protocols wayland libva-utils libva-nvidia-driver libglvnd s2tc spirv-headers spirv-tools libclc glslang util-macros x11-app xbitmaps xkeyboard-config libxkbcommon pixman libepoxy x11-font libxcvt xorg-server libinput xf86-input-libinput xf86-input-vmmouse xf86-input-wacom xf86-video-amdgpu xf86-video-ati xf86-video-dummy xf86-video-qxl xf86-video-r128 xf86-video-vesa xf86-video-vmware vulkan-headers vulkan-loader vulkan-utility-libraries vulkan-validationlayers vulkan glu glfw freeglut egl-wayland xwayland libdisplay-info aml neatvnc weston vkd3d tinyx xcursor-themes xinit xsettingsd peg intel-gpu-tools glmark2 mesa-demos
```

Test Build(s) Done
------------------

**Primary Architectures**

<!-- Please remove any architecture to which this topic does not apply. -->

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`
<!-- If this package involves a `+32` counterpart, please uncomment the line below. -->    
<!-- - [ ] 32-bit Optional Environment `optenv32` -->

<!-- If all package(s) affected by this topic is `noarch`, please use the stub below. -->
<!-- - [ ] Architecture-independent `noarch` -->

**Secondary Architectures**

<!-- Please remove any architecture to which this topic does not apply. -->

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [ ] MIPS R6 64-bit (Little Endian) `mips64r6el`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

<!-- Maintainers should review file changes and make sure that the change(s) made complies with our [Package Styling Manual](https://wiki.aosc.io/developer/packaging/package-styling-manual/) -->

<!-- Maintainers and users may now test the packages in this topic and, once user/maintainer feedback indicates that the update(s) work as expected and find its quality satisfactory,
     another maintainer may now review this pull request and mark it as Approved. After which, the maintainer will build affected package(s) and upload them to the `stable` repository. -->

RedL0tus committed:
xwayland: update to 23.2.2

- Use ABTYPE=meson.

RedL0tus committed:
libxdmcp: update to 1.1.4

RedL0tus committed:
libxau: update to 1.0.11

RedL0tus committed:
xcb-proto: update to 1.16.0

RedL0tus committed:
x11-proto: update to 2023.2

RedL0tus committed:
libxcb: update to 1.16

- Set PYTHON to /usr/bin/python3

RedL0tus committed:
xcb-util-image: update to 0.4.1

RedL0tus committed:
xcb-util-wm: update to 0.4.2

RedL0tus committed:
xcb-util-cursor: update to 0.1.4

RedL0tus committed:
xcb-util-keysyms: update to 0.4.1

RedL0tus committed:
x11-lib: update to 7.7.2020231031

- Update xtrans to 1.5.0.
- Update libX11 to 1.8.7.
- Update libXext to 1.3.5.
- Update libFS to 1.0.9.
- Update libICE to 1.1.1.
- Update libSM to 1.2.4.
- Update libXScrnSaver to 1.2.4.
- Update libXt to 1.3.0.
- Update libXmu to 1.1.4.
- Update libXpm to 3.5.17.
- Update libXaw to 1.0.15.
- Update libXfixes to 6.0.1.
- Update libXcomposite to 0.4.6.
- Update libXrender to 0.9.11.
- Update libXdamage to 1.1.6.
- Update libfontenc to 1.1.7.
- Update libXfont2 to 2.0.6.
- Update libXft to 2.3.8.
- Update libXi to 1.8.1.
- Update libXinerama to 1.1.5.
- Update libXrandr to 1.5.4.
- Update libXres to 1.2.2.
- Update libXtst to 1.2.4.
- Update libXv to 1.0.12.
- Update libXxf86dga to 1.1.6.
- Update libXxf86vm to 1.1.5.
- Update libdmx to 1.1.5.
- Update libpciaccess to 0.17.
- Update libxkbfile to 1.1.2.
- Update libXp to 1.0.4.
- Update libXpresent to 1.0.1.

RedL0tus committed:
xcb-util: update to 0.4.1

RedL0tus committed:
xcb-util-renderutil: update to 0.3.10

RedL0tus committed:
libglvnd: update to 1.7.0

RedL0tus committed:
libva-nvidia-driver: update to 0.0.10

RedL0tus committed:
libvdpau: update to 1.5

RedL0tus committed:
wayland-protocols: update to 1.32

RedL0tus committed:
libva: update to 2.20.0

RedL0tus committed:
wayland: update to 1.22.0

RedL0tus committed:
libva-utils: update to 2.20.0

RedL0tus committed:
libdrm: update to 2.4.117

- Explicitly enable libdrm_intel by default.

RedL0tus committed:
spirv-headers: update to 1.3.268.0

RedL0tus committed:
x11-app: update to 7.7.20231031

- Update spec file to let acbs itself handle downloads.
- Use version-agnostic build script.
- Update iceauth to 1.0.9.
- Update mkfontscale to 1.2.2.
- Update sessreg to 1.1.3.
- Update setxkbmap to 1.3.4.
- Update smproxy to 1.0.7.
- Update xcmsdb to 1.0.6.
- Update xcursorgen to 1.0.8.
- Update xdpyinfo to 1.3.2.
- Update xdriinfo to 1.0.7.
- Update xev to 1.2.5.
- Update xgamma to 1.0.7.
- Update xkbcomp to 1.4.6.
- Update xkbevd to 1.1.5.
- Update xkbutils 1.0.5.
- Update xkill to 1.0.6.
- Update xlsatoms 1.1.4.
- Update xlsclients to 1.1.5.
- Update xmessage to 1.0.6.
- Update xmodmap to 10.0.11.
- Update xpr to 1.0.6.
- Update xprop to 1.1.0.
- Update xrandr to 1.5.2.
- Update xrdb to 1.2.2.
- Update xrefresh to 1.0.7.
- Update xset to 1.2.5.
- Update xsetroot to 1.1.3.
- Update xvinfo to 1.1.5.
- Update xwd to 1.0.9.
- Update xwininfo to 1.1.6.
- UPdate xwud to 1.0.6.

RedL0tus committed:
util-macros: update to 1.20.0

RedL0tus committed:
xbitmaps: update to 1.1.3

RedL0tus committed:
libclc: update to 17.0.3

RedL0tus committed:
glslang: update to 13.1.1

- Add SPIRV-Tools to BUILDDEP.
- Enable ALLOW_EXTERNAL_SPIRV_TOOLS to use system SPIRV-Tools.

RedL0tus committed:
s2tc: update to 1.0+git20210317

RedL0tus committed:
pixman: update to 0.42.2

RedL0tus committed:
libxkbcommon: update to 1.6.0

RedL0tus committed:
libepoxy: update to 1.5.10

RedL0tus committed:
x11-font: update to 7.7.20231106

- Update spec file to let acbs itself handle downloads.
- Use version-agnostic build script.
- Update font-util to 1.4.1.
- Update encodings to 1.0.7.
- Update font-adobe-100dpi to 1.0.4.
- Update font-adobe-75dpi to 1.0.4.
- Update font-adobe-utopia-100dpi to 1.0.5.
- Update font-adobe-utopia-75dpi to 1.0.5.
- Update font-adobe-utopia-type1 to 1.0.5.
- Update font-alias to 1.0.5.
- Update font-arabic-misc to 1.0.4.
- Update font-bh-100dpi to 1.0.4.
- Update font-bh-75dpi to 1.0.4.
- Update font-bh-lucidatypewriter-100dpi to 1.0.4.
- Update font-bh-lucidatypewriter-75dpi to 1.0.4.
- Update font-bh-ttf to 1.0.4.
- Update font-bh-type1 to 1.0.4.
- Update font-bitstream-100dpi to 1.0.4.
- Update font-bitstream-75dpi to 1.0.4.
- Update font-bitstream-type1 to 1.0.4.
- Update font-cronyx-cyrillic to 1.0.4.
- Update font-cursor-misc to 1.0.4.
- Update font-daewoo-misc to 1.0.4.
- Update font-dec-misc to 1.0.4.
- Update font-ibm-type1 to 1.0.4.
- Update font-isas-misc to 1.0.4.
- Update font-jis-misc to 1.0.4.
- Update font-micro-misc to 1.0.4.
- Update font-misc-cyrillic to 1.0.4.
- Update font-misc-ethiopic to 1.0.5.
- Update font-misc-meltho to 1.0.4.
- Update font-misc-misc to 1.1.3.
- Update font-mutt-misc to 1.0.4.
- Update font-schumacher-misc to 1.1.3.
- Update font-screen-cyrillic to 1.0.5.
- Update font-sony-misc to 1.0.4.
- Update font-sun-misc to 1.0.4.
- Update font-winitzki-cyrillic to 1.0.4.
- Update font-xfree86-type1 to 1.0.5.

RedL0tus committed:
xkeyboard-config: update to 2.40

RedL0tus committed:
libxcvt: update to 0.1.2

RedL0tus committed:
xorg-server: update to 21.1.9

RedL0tus committed:
libinput: update to 1.24.0

RedL0tus committed:
xf86-input-libinput: update to 1.4.0

RedL0tus committed:
xf86-input-vmmouse: update to 13.2.0

RedL0tus committed:
xf86-input-wacom: update to 13.4.0

RedL0tus committed:
xf86-video-amdgpu: update to 23.0.0

RedL0tus committed:
xf86-video-ati: update to 22.0.0

RedL0tus committed:
xf86-video-dummy: update to 0.4.1

RedL0tus committed:
xf86-video-qxl: update to 0.1.6

- Remove outdated patches.

RedL0tus committed:
xf86-video-r128: update to 6.12.1

RedL0tus committed:
xf86-video-vesa: update to 2.6.0

RedL0tus committed:
xf86-video-vmware: update to 13.4.0

- Remove outdated patch.

RedL0tus committed:
vulkan-headers: update to 1.3.269

RedL0tus committed:
vulkan-loader: update to 1.3.269

RedL0tus committed:
vulkan-utility-libraries: new, 1.3.269

RedL0tus committed:
spirv-tools: update to 2023.5~rc1

- Remove outdated patchset.

RedL0tus committed:
vulkan-validationlayers: update to 1.3.269

RedL0tus committed:
glu: update to 9.0.3

RedL0tus committed:
glfw: update to 3.3.8

- Fix docs installation.

RedL0tus committed:
freeglut: update to 3.4.0

RedL0tus committed:
egl-wayland: update to 1.1.13

RedL0tus committed:
libdisplay-info: new, 0.1.1

RedL0tus committed:
aml: new, 0.3.0

RedL0tus committed:
neatvnc: new, 0.7.1

RedL0tus committed:
weston: update to 12.0.2

- Backport patch for neatvnc 0.7 compatibility.

RedL0tus committed:
vkd3d: update to 1.9

RedL0tus committed:
tinyx: update to 1.3.0+git20221106

- Add patches for AArch64, RISC-V and LoongArch64 support.

RedL0tus committed:
xcursor-themes: update to 1.0.7

RedL0tus committed:
xinit: update to 1.4.2

RedL0tus committed:
xsettingsd: update to 1.0.2

MingcongBai commented on app-imaging/glslang/autobuild/defines:
```diff
@@ -1,7 +1,10 @@
 PKGSEC=graphics
-PKGDEP="python-2 glibc"
-BUILDDEP="bison"
+PKGDEP="python-3 glibc"
```
`python-3` should probably be in BUILDDEP instead, there is no Python component nor ELF/Shebang dependency on it.

RedL0tus replied:
Done.

MingcongBai replied:
> Done.

I don't see that you've pushed any changes?

MingcongBai commented on app-imaging/glslang/autobuild/defines:
```diff
@@ -1,7 +1,10 @@
-PKGDEP="python-2 glibc"
-BUILDDEP="bison"
+PKGDEP="python-3 glibc"
+BUILDDEP="bison cmake ninja spirv-tools"
```
`bison cmake ninja` are not necessary.

RedL0tus replied:
Fixed.

MingcongBai commented on runtime-display/mesa/spec:
```diff
@@ -1,4 +1,5 @@
 VER=23.2.1
+REL=1
```
What changed?

MingcongBai commented on runtime-display/vulkan-utility-libraries/autobuild/defines:
```diff
@@ -0,0 +1,7 @@
+PKGNAME=vulkan-utility-libraries
```
Add this as a dep for `vulkan`.

RedL0tus replied:
Done.

MingcongBai commented on runtime-display/vulkan-utility-libraries/spec:
```diff
@@ -0,0 +1,3 @@
+VER=1.3.269
+SRCS="tbl::https://github.com/KhronosGroup/Vulkan-Utility-Libraries/archive/refs/tags/v$VER.tar.gz"
+CHKSUMS="sha256::1ba3a9937aa2932c8d3ed19949df95612ca1a4d5f64b892c8a1559a03bc355b1"
```
Missing CHKUPDATE.

RedL0tus replied:
Added.

MingcongBai commented on runtime-display/weston/spec:
```diff
@@ -1,4 +1,4 @@
-VER=10.0.2
+VER=12.0.93
```
.93 is an unstable version (RC1 according to upstream), use 12.0.2 instead.

RedL0tus replied:
Fixed.

MingcongBai commented on runtime-display/x11-lib/autobuild/build-order:
```diff
@@ -1,35 +1,35 @@
-xtrans-1.4.0.tar.gz
```
Consider rewriting this as you would for x11-app/font?

RedL0tus replied:
Done.

MingcongBai commented on runtime-display/xwayland/autobuild/beyond:
```diff
@@ -0,0 +1,4 @@
+# xorg-server also provides these files
+abinfo "Removing duplicated files ..."
+rm -rvf "$PKGDIR/usr/lib/xorg"
```
`-r` is not necessary here and `-f` should not be used. Also consider merging the two commands.

RedL0tus replied:
Fixed.

MingcongBai commented on runtime-network/aml/autobuild/defines:
```diff
@@ -0,0 +1,5 @@
+PKGNAME=aml
+PKGSEC=libs
+PKGDES="Andri's Main Loop"
```
What does that mean? Please be more descriptive.

RedL0tus replied:
Fixed.

MingcongBai commented on runtime-network/aml/autobuild/defines:
```diff
@@ -0,0 +1,5 @@
+PKGSEC=libs
+PKGDES="Andri's Main Loop"
+PKGDEP="glibc"
+BUILDDEP="meson"
```
This is not necessary.

RedL0tus replied:
Fixed.

MingcongBai commented on runtime-network/aml/spec:
```diff
@@ -0,0 +1,4 @@
+VER=0.3.0
+SRCS="https://github.com/any1/aml/archive/refs/tags/v$VER.tar.gz"
```
Please try not to use automatically packaged tarballs, use Git tags instead.

RedL0tus replied:
Fixed.

MingcongBai commented on runtime-network/neatvnc/autobuild/defines:
```diff
@@ -0,0 +1,5 @@
+PKGNAME=neatvnc
+PKGSEC=net
+PKGDES="A liberally licensed VNC server library with a clean interface"
```
This means very little to the end user - try this:

```
A fast VNC server library with a clean API interface
```

RedL0tus replied:
Fixed.

MingcongBai commented on runtime-network/neatvnc/autobuild/defines:
```diff
@@ -0,0 +1,5 @@
+PKGSEC=net
+PKGDES="A liberally licensed VNC server library with a clean interface"
+PKGDEP="aml ffmpeg mesa gnutls libdrm libjpeg-turbo gmp pixman zlib"
+BUILDDEP="meson"
```
Not necessary.

RedL0tus replied:
Removed.

MingcongBai commented on runtime-network/neatvnc/spec:
```diff
@@ -0,0 +1,4 @@
+VER=0.7.1
+SRCS="https://github.com/any1/neatvnc/archive/refs/tags/v$VER.tar.gz"
```
Please try not to use automatically packaged tarballs, use Git tags instead.

RedL0tus replied:
Fixed.

MingcongBai commented on runtime-multimedia/libva-nvidia-driver/spec:
```diff
@@ -1,5 +1,4 @@
-SRCS="tbl::https://github.com/elFarto/nvidia-vaapi-driver/archive/refs/tags/v0.0.6.tar.gz"
-CHKSUMS="sha256::8a64c069200750f12d013c79547e459cbb87b498357478c5f36cb97f710294e0"
+VER=0.0.10
+SRCS="tbl::https://github.com/elFarto/nvidia-vaapi-driver/archive/refs/tags/v$VER.tar.gz"
```
Please try not to use automatically packaged tarballs, use Git tags instead.

RedL0tus replied:
Fixed.

MingcongBai commented on runtime-display/vulkan-validationlayers/spec:
```diff
@@ -1,4 +1,4 @@
-SRCS="tbl::https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/sdk-$VER.tar.gz"
-CHKSUMS="sha256::08e308c13a762134d0a66a66de8cd5a32e4e73cfe6006247292570cc5b2b4b0b"
+VER=1.3.269
+SRCS="tbl::https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/v$VER.tar.gz"
```
Please try not to use automatically packaged tarballs, use Git tags instead.

RedL0tus replied:
Fixed.

MingcongBai commented on runtime-display/vulkan-utility-libraries/spec:
```diff
@@ -0,0 +1,3 @@
+VER=1.3.269
+SRCS="tbl::https://github.com/KhronosGroup/Vulkan-Utility-Libraries/archive/refs/tags/v$VER.tar.gz"
```
Please try not to use automatically packaged tarballs, use Git tags instead.

RedL0tus replied:
Fixed.

MingcongBai commented on runtime-display/libdisplay-info/spec:
```diff
@@ -0,0 +1,4 @@
+VER=0.1.1
+SRCS="https://gitlab.freedesktop.org/emersion/libdisplay-info/-/archive/$VER/libdisplay-info-$VER.tar.gz"
```
Please try not to use automatically packaged tarballs, use Git tags instead.

RedL0tus replied:
Fixed.

MingcongBai commented on runtime-display/glfw/spec:
```diff
@@ -1,5 +1,4 @@
-VER=3.3.2
-REL=1
+VER=3.3.8
 SRCS="tbl::https://github.com/glfw/glfw/releases/download/$VER/glfw-$VER.zip"
```
Please try not to use automatically packaged tarballs, use Git tags instead.

RedL0tus replied:
Fixed.

MingcongBai commented on runtime-display/egl-wayland/spec:
```diff
@@ -1,4 +1,4 @@
-VER=1.1.7
+VER=1.1.13
 SRCS="tbl::https://github.com/NVIDIA/egl-wayland/archive/$VER.tar.gz"
```
Please try not to use automatically packaged tarballs, use Git tags instead.

RedL0tus replied:
Fixed.

MingcongBai commented on runtime-devices/libinput/spec:
```diff
@@ -1,5 +1,4 @@
-VER=1.21.0
-REL=1
+VER=1.24.0
 SRCS="tbl::https://gitlab.freedesktop.org/libinput/libinput/-/archive/$VER/libinput-$VER.tar.gz"
```
Please try not to use automatically packaged tarballs, use Git tags instead.

RedL0tus replied:
Fixed.

MingcongBai commented on app-imaging/glslang/spec:
```diff
@@ -1,4 +1,4 @@
-VER=11.11.0
+VER=13.1.1
 SRCS="tbl::https://github.com/KhronosGroup/glslang/archive/$VER.tar.gz"
```
Please try not to use automatically packaged tarballs, use Git tags instead.

RedL0tus replied:
Fixed.

MingcongBai commented on runtime-display/s2tc/spec:
```diff
@@ -1,5 +1,4 @@
-VER=20151227
-REL=3
-SRCS="git::commit=f6ec862d7594e29ae80a6e49f66ad4c76cf09abc::https://github.com/divVerent/s2tc"
+VER=20210317
```
The version should be `1.0+git20210317` according to the Styling Manual.

RedL0tus replied:
Fixed.

MingcongBai commented on runtime-display/vulkan-headers/spec:
```diff
@@ -1,4 +1,4 @@
-VER=1.3.236
+VER=1.3.269
 SRCS="tbl::https://github.com/KhronosGroup/Vulkan-Headers/archive/v$VER.tar.gz"
```
Please try not to use automatically packaged tarballs, use Git tags instead.

RedL0tus replied:
Fixed.

MingcongBai commented on runtime-display/vulkan-loader/spec:
```diff
@@ -1,4 +1,4 @@
-VER=1.3.236
+VER=1.3.269
 SRCS="tbl::https://github.com/KhronosGroup/Vulkan-Loader/archive/v$VER.tar.gz"
```
Please try not to use automatically packaged tarballs, use Git tags instead.

RedL0tus replied:
Fixed.

MingcongBai commented:
Also, the build order contains `xf86-video-intel` but there was no update.

MingcongBai commented:
Please also update `intel-gpu-tools`, `glmark2`, and `mesa-demos`.

MingcongBai commented:
```
mingcongbai@JellyNote [ 视频 ] $  mpv *.mkv
mpv: error while loading shared libraries: libSPIRV.so: cannot open shared object file: No such file or directory
```

MingcongBai commented:
Is this normal?

```
EGL client extensions string:
    EGL_EXT_device_base EGL_EXT_device_enumeration EGL_EXT_device_query
    EGL_EXT_platform_base EGL_KHR_client_get_all_proc_addresses
    EGL_EXT_client_extensions EGL_KHR_debug EGL_EXT_platform_device
    EGL_EXT_platform_wayland EGL_KHR_platform_wayland
    EGL_EXT_platform_x11 EGL_KHR_platform_x11 EGL_EXT_platform_xcb
    EGL_MESA_platform_gbm EGL_KHR_platform_gbm
    EGL_MESA_platform_surfaceless

GBM platform:
EGL API version: 1.5
EGL vendor string: Mesa Project
EGL version string: 1.5
EGL client APIs: OpenGL OpenGL_ES 
EGL extensions string:
    EGL_ANDROID_blob_cache EGL_ANDROID_native_fence_sync
    EGL_EXT_buffer_age EGL_EXT_create_context_robustness
    EGL_EXT_image_dma_buf_import EGL_EXT_image_dma_buf_import_modifiers
    EGL_EXT_protected_content EGL_IMG_context_priority EGL_KHR_cl_event2
    EGL_KHR_config_attribs EGL_KHR_context_flush_control
    EGL_KHR_create_context EGL_KHR_create_context_no_error
    EGL_KHR_fence_sync EGL_KHR_get_all_proc_addresses
    EGL_KHR_gl_colorspace EGL_KHR_gl_renderbuffer_image
    EGL_KHR_gl_texture_2D_image EGL_KHR_gl_texture_3D_image
    EGL_KHR_gl_texture_cubemap_image EGL_KHR_image EGL_KHR_image_base
    EGL_KHR_image_pixmap EGL_KHR_no_config_context EGL_KHR_reusable_sync
    EGL_KHR_surfaceless_context EGL_EXT_pixel_format_float
    EGL_KHR_wait_sync EGL_MESA_configless_context EGL_MESA_drm_image
    EGL_MESA_image_dma_buf_export EGL_MESA_query_driver
    EGL_WL_bind_wayland_display
Configurations:
     bf lv colorbuffer dp st  ms    vis   cav bi  renderable  supported
  id sz  l  r  g  b  a th cl ns b    id   eat nd gl es es2 vg surfaces 
---------------------------------------------------------------------
0x01 32  0 10 10 10  2  0  0  0 0 0x30335241--         y  y  y     win
0x02 32  0 10 10 10  2 16  0  0 0 0x30335241--         y  y  y     win
0x03 32  0 10 10 10  2 24  0  0 0 0x30335241--         y  y  y     win
0x04 32  0 10 10 10  2 24  8  0 0 0x30335241--         y  y  y     win
0x05 32  0 10 10 10  2  0  0  2 1 0x30335241--         y  y  y     win
0x06 32  0 10 10 10  2  0  0  4 1 0x30335241--         y  y  y     win
0x07 32  0 10 10 10  2  0  0  8 1 0x30335241--         y  y  y     win
0x08 32  0 10 10 10  2  0  0 16 1 0x30335241--         y  y  y     win
0x09 32  0 10 10 10  2 16  0  2 1 0x30335241--         y  y  y     win
0x0a 32  0 10 10 10  2 16  0  4 1 0x30335241--         y  y  y     win
0x0b 32  0 10 10 10  2 16  0  8 1 0x30335241--         y  y  y     win
0x0c 32  0 10 10 10  2 16  0 16 1 0x30335241--         y  y  y     win
0x0d 32  0 10 10 10  2 24  0  2 1 0x30335241--         y  y  y     win
0x0e 32  0 10 10 10  2 24  0  4 1 0x30335241--         y  y  y     win
0x0f 32  0 10 10 10  2 24  0  8 1 0x30335241--         y  y  y     win
0x10 32  0 10 10 10  2 24  0 16 1 0x30335241--         y  y  y     win
0x11 32  0 10 10 10  2 24  8  2 1 0x30335241--         y  y  y     win
0x12 32  0 10 10 10  2 24  8  4 1 0x30335241--         y  y  y     win
0x13 32  0 10 10 10  2 24  8  8 1 0x30335241--         y  y  y     win
0x14 32  0 10 10 10  2 24  8 16 1 0x30335241--         y  y  y     win
0x15 30  0 10 10 10  0  0  0  0 0 0x30335258--         y  y  y     win
0x16 30  0 10 10 10  0 16  0  0 0 0x30335258--         y  y  y     win
0x17 30  0 10 10 10  0 24  0  0 0 0x30335258--         y  y  y     win
0x18 30  0 10 10 10  0 24  8  0 0 0x30335258--         y  y  y     win
0x19 30  0 10 10 10  0  0  0  2 1 0x30335258--         y  y  y     win
0x1a 30  0 10 10 10  0  0  0  4 1 0x30335258--         y  y  y     win
0x1b 30  0 10 10 10  0  0  0  8 1 0x30335258--         y  y  y     win
0x1c 30  0 10 10 10  0  0  0 16 1 0x30335258--         y  y  y     win
0x1d 30  0 10 10 10  0 16  0  2 1 0x30335258--         y  y  y     win
0x1e 30  0 10 10 10  0 16  0  4 1 0x30335258--         y  y  y     win
0x1f 30  0 10 10 10  0 16  0  8 1 0x30335258--         y  y  y     win
0x20 30  0 10 10 10  0 16  0 16 1 0x30335258--         y  y  y     win
0x21 30  0 10 10 10  0 24  0  2 1 0x30335258--         y  y  y     win
0x22 30  0 10 10 10  0 24  0  4 1 0x30335258--         y  y  y     win
0x23 30  0 10 10 10  0 24  0  8 1 0x30335258--         y  y  y     win
0x24 30  0 10 10 10  0 24  0 16 1 0x30335258--         y  y  y     win
0x25 30  0 10 10 10  0 24  8  2 1 0x30335258--         y  y  y     win
0x26 30  0 10 10 10  0 24  8  4 1 0x30335258--         y  y  y     win
0x27 30  0 10 10 10  0 24  8  8 1 0x30335258--         y  y  y     win
0x28 30  0 10 10 10  0 24  8 16 1 0x30335258--         y  y  y     win
0x29 32  0  8  8  8  8  0  0  0 0 0x34325241--         y  y  y     win
0x2a 32  0  8  8  8  8 16  0  0 0 0x34325241--         y  y  y     win
0x2b 32  0  8  8  8  8 24  0  0 0 0x34325241--         y  y  y     win
0x2c 32  0  8  8  8  8 24  8  0 0 0x34325241--         y  y  y     win
0x2d 32  0  8  8  8  8  0  0  2 1 0x34325241--         y  y  y     win
0x2e 32  0  8  8  8  8  0  0  4 1 0x34325241--         y  y  y     win
0x2f 32  0  8  8  8  8  0  0  8 1 0x34325241--         y  y  y     win
0x30 32  0  8  8  8  8  0  0 16 1 0x34325241--         y  y  y     win
0x31 32  0  8  8  8  8 16  0  2 1 0x34325241--         y  y  y     win
0x32 32  0  8  8  8  8 16  0  4 1 0x34325241--         y  y  y     win
0x33 32  0  8  8  8  8 16  0  8 1 0x34325241--         y  y  y     win
0x34 32  0  8  8  8  8 16  0 16 1 0x34325241--         y  y  y     win
0x35 32  0  8  8  8  8 24  0  2 1 0x34325241--         y  y  y     win
0x36 32  0  8  8  8  8 24  0  4 1 0x34325241--         y  y  y     win
0x37 32  0  8  8  8  8 24  0  8 1 0x34325241--         y  y  y     win
0x38 32  0  8  8  8  8 24  0 16 1 0x34325241--         y  y  y     win
0x39 32  0  8  8  8  8 24  8  2 1 0x34325241--         y  y  y     win
0x3a 32  0  8  8  8  8 24  8  4 1 0x34325241--         y  y  y     win
0x3b 32  0  8  8  8  8 24  8  8 1 0x34325241--         y  y  y     win
0x3c 32  0  8  8  8  8 24  8 16 1 0x34325241--         y  y  y     win
0x3d 24  0  8  8  8  0  0  0  0 0 0x34325258--         y  y  y     win
0x3e 24  0  8  8  8  0 16  0  0 0 0x34325258--         y  y  y     win
0x3f 24  0  8  8  8  0 24  0  0 0 0x34325258--         y  y  y     win
0x40 24  0  8  8  8  0 24  8  0 0 0x34325258--         y  y  y     win
0x41 24  0  8  8  8  0  0  0  2 1 0x34325258--         y  y  y     win
0x42 24  0  8  8  8  0  0  0  4 1 0x34325258--         y  y  y     win
0x43 24  0  8  8  8  0  0  0  8 1 0x34325258--         y  y  y     win
0x44 24  0  8  8  8  0  0  0 16 1 0x34325258--         y  y  y     win
0x45 24  0  8  8  8  0 16  0  2 1 0x34325258--         y  y  y     win
0x46 24  0  8  8  8  0 16  0  4 1 0x34325258--         y  y  y     win
0x47 24  0  8  8  8  0 16  0  8 1 0x34325258--         y  y  y     win
0x48 24  0  8  8  8  0 16  0 16 1 0x34325258--         y  y  y     win
0x49 24  0  8  8  8  0 24  0  2 1 0x34325258--         y  y  y     win
0x4a 24  0  8  8  8  0 24  0  4 1 0x34325258--         y  y  y     win
0x4b 24  0  8  8  8  0 24  0  8 1 0x34325258--         y  y  y     win
0x4c 24  0  8  8  8  0 24  0 16 1 0x34325258--         y  y  y     win
0x4d 24  0  8  8  8  0 24  8  2 1 0x34325258--         y  y  y     win
0x4e 24  0  8  8  8  0 24  8  4 1 0x34325258--         y  y  y     win
0x4f 24  0  8  8  8  0 24  8  8 1 0x34325258--         y  y  y     win
0x50 24  0  8  8  8  0 24  8 16 1 0x34325258--         y  y  y     win
0x51 16  0  5  6  5  0  0  0  0 0 0x36314752--         y  y  y     win
0x52 16  0  5  6  5  0 16  0  0 0 0x36314752--         y  y  y     win
0x53 16  0  5  6  5  0 24  0  0 0 0x36314752--         y  y  y     win
0x54 16  0  5  6  5  0 24  8  0 0 0x36314752--         y  y  y     win
0x55 16  0  5  6  5  0  0  0  2 1 0x36314752--         y  y  y     win
0x56 16  0  5  6  5  0  0  0  4 1 0x36314752--         y  y  y     win
0x57 16  0  5  6  5  0  0  0  8 1 0x36314752--         y  y  y     win
0x58 16  0  5  6  5  0  0  0 16 1 0x36314752--         y  y  y     win
0x59 16  0  5  6  5  0 16  0  2 1 0x36314752--         y  y  y     win
0x5a 16  0  5  6  5  0 16  0  4 1 0x36314752--         y  y  y     win
0x5b 16  0  5  6  5  0 16  0  8 1 0x36314752--         y  y  y     win
0x5c 16  0  5  6  5  0 16  0 16 1 0x36314752--         y  y  y     win
0x5d 16  0  5  6  5  0 24  0  2 1 0x36314752--         y  y  y     win
0x5e 16  0  5  6  5  0 24  0  4 1 0x36314752--         y  y  y     win
0x5f 16  0  5  6  5  0 24  0  8 1 0x36314752--         y  y  y     win
0x60 16  0  5  6  5  0 24  0 16 1 0x36314752--         y  y  y     win
0x61 16  0  5  6  5  0 24  8  2 1 0x36314752--         y  y  y     win
0x62 16  0  5  6  5  0 24  8  4 1 0x36314752--         y  y  y     win
0x63 16  0  5  6  5  0 24  8  8 1 0x36314752--         y  y  y     win
0x64 16  0  5  6  5  0 24  8 16 1 0x36314752--         y  y  y     win
0x65 64  0 16 16 16 16  0  0  0 0 0x48344241--         y  y  y     win
0x66 64  0 16 16 16 16 16  0  0 0 0x48344241--         y  y  y     win
0x67 64  0 16 16 16 16 24  0  0 0 0x48344241--         y  y  y     win
0x68 64  0 16 16 16 16 24  8  0 0 0x48344241--         y  y  y     win
0x69 64  0 16 16 16 16  0  0  2 1 0x48344241--         y  y  y     win
0x6a 64  0 16 16 16 16  0  0  4 1 0x48344241--         y  y  y     win
0x6b 64  0 16 16 16 16  0  0  8 1 0x48344241--         y  y  y     win
0x6c 64  0 16 16 16 16  0  0 16 1 0x48344241--         y  y  y     win
0x6d 64  0 16 16 16 16 16  0  2 1 0x48344241--         y  y  y     win
0x6e 64  0 16 16 16 16 16  0  4 1 0x48344241--         y  y  y     win
0x6f 64  0 16 16 16 16 16  0  8 1 0x48344241--         y  y  y     win
0x70 64  0 16 16 16 16 16  0 16 1 0x48344241--         y  y  y     win
0x71 64  0 16 16 16 16 24  0  2 1 0x48344241--         y  y  y     win
0x72 64  0 16 16 16 16 24  0  4 1 0x48344241--         y  y  y     win
0x73 64  0 16 16 16 16 24  0  8 1 0x48344241--         y  y  y     win
0x74 64  0 16 16 16 16 24  0 16 1 0x48344241--         y  y  y     win
0x75 64  0 16 16 16 16 24  8  2 1 0x48344241--         y  y  y     win
0x76 64  0 16 16 16 16 24  8  4 1 0x48344241--         y  y  y     win
0x77 64  0 16 16 16 16 24  8  8 1 0x48344241--         y  y  y     win
0x78 64  0 16 16 16 16 24  8 16 1 0x48344241--         y  y  y     win
0x79 48  0 16 16 16  0  0  0  0 0 0x48344258--         y  y  y     win
0x7a 48  0 16 16 16  0 16  0  0 0 0x48344258--         y  y  y     win
0x7b 48  0 16 16 16  0 24  0  0 0 0x48344258--         y  y  y     win
0x7c 48  0 16 16 16  0 24  8  0 0 0x48344258--         y  y  y     win
0x7d 48  0 16 16 16  0  0  0  2 1 0x48344258--         y  y  y     win
0x7e 48  0 16 16 16  0  0  0  4 1 0x48344258--         y  y  y     win
0x7f 48  0 16 16 16  0  0  0  8 1 0x48344258--         y  y  y     win
0x80 48  0 16 16 16  0  0  0 16 1 0x48344258--         y  y  y     win
0x81 48  0 16 16 16  0 16  0  2 1 0x48344258--         y  y  y     win
0x82 48  0 16 16 16  0 16  0  4 1 0x48344258--         y  y  y     win
0x83 48  0 16 16 16  0 16  0  8 1 0x48344258--         y  y  y     win
0x84 48  0 16 16 16  0 16  0 16 1 0x48344258--         y  y  y     win
0x85 48  0 16 16 16  0 24  0  2 1 0x48344258--         y  y  y     win
0x86 48  0 16 16 16  0 24  0  4 1 0x48344258--         y  y  y     win
0x87 48  0 16 16 16  0 24  0  8 1 0x48344258--         y  y  y     win
0x88 48  0 16 16 16  0 24  0 16 1 0x48344258--         y  y  y     win
0x89 48  0 16 16 16  0 24  8  2 1 0x48344258--         y  y  y     win
0x8a 48  0 16 16 16  0 24  8  4 1 0x48344258--         y  y  y     win
0x8b 48  0 16 16 16  0 24  8  8 1 0x48344258--         y  y  y     win
0x8c 48  0 16 16 16  0 24  8 16 1 0x48344258--         y  y  y     win

Wayland platform:
eglinfo: eglInitialize failed

X11 platform:
EGL API version: 1.5
EGL vendor string: Mesa Project
EGL version string: 1.5
EGL client APIs: OpenGL OpenGL_ES 
EGL extensions string:
    EGL_ANDROID_blob_cache EGL_ANDROID_native_fence_sync
    EGL_CHROMIUM_sync_control EGL_ANGLE_sync_control_rate
    EGL_EXT_buffer_age EGL_EXT_create_context_robustness
    EGL_EXT_image_dma_buf_import EGL_EXT_image_dma_buf_import_modifiers
    EGL_EXT_protected_content EGL_EXT_swap_buffers_with_damage
    EGL_IMG_context_priority EGL_KHR_cl_event2 EGL_KHR_config_attribs
    EGL_KHR_context_flush_control EGL_KHR_create_context
    EGL_KHR_create_context_no_error EGL_KHR_fence_sync
    EGL_KHR_get_all_proc_addresses EGL_KHR_gl_colorspace
    EGL_KHR_gl_renderbuffer_image EGL_KHR_gl_texture_2D_image
    EGL_KHR_gl_texture_3D_image EGL_KHR_gl_texture_cubemap_image
    EGL_KHR_image EGL_KHR_image_base EGL_KHR_image_pixmap
    EGL_KHR_no_config_context EGL_KHR_reusable_sync
    EGL_KHR_surfaceless_context EGL_KHR_swap_buffers_with_damage
    EGL_EXT_pixel_format_float EGL_KHR_wait_sync
    EGL_MESA_configless_context EGL_MESA_drm_image
    EGL_MESA_image_dma_buf_export EGL_MESA_query_driver
    EGL_NOK_texture_from_pixmap EGL_WL_bind_wayland_display
Configurations:
     bf lv colorbuffer dp st  ms    vis   cav bi  renderable  supported
  id sz  l  r  g  b  a th cl ns b    id   eat nd gl es es2 vg surfaces 
---------------------------------------------------------------------
0x01 32  0  8  8  8  8  0  0  0 0 0x21TC      a  y  y  y     win,pb,pix
0x02 32  0  8  8  8  8 16  0  0 0 0x21TC      a  y  y  y     win,pb,pix
0x03 32  0  8  8  8  8 24  0  0 0 0x21TC      a  y  y  y     win,pb,pix
0x04 32  0  8  8  8  8 24  8  0 0 0x21TC      a  y  y  y     win,pb,pix
0x05 32  0  8  8  8  8  0  0  2 1 0x21TC      a  y  y  y     win,pb,pix
0x06 32  0  8  8  8  8  0  0  4 1 0x21TC      a  y  y  y     win,pb,pix
0x07 32  0  8  8  8  8  0  0  8 1 0x21TC      a  y  y  y     win,pb,pix
0x08 32  0  8  8  8  8  0  0 16 1 0x21TC      a  y  y  y     win,pb,pix
0x09 32  0  8  8  8  8 16  0  2 1 0x21TC      a  y  y  y     win,pb,pix
0x0a 32  0  8  8  8  8 16  0  4 1 0x21TC      a  y  y  y     win,pb,pix
0x0b 32  0  8  8  8  8 16  0  8 1 0x21TC      a  y  y  y     win,pb,pix
0x0c 32  0  8  8  8  8 16  0 16 1 0x21TC      a  y  y  y     win,pb,pix
0x0d 32  0  8  8  8  8 24  0  2 1 0x21TC      a  y  y  y     win,pb,pix
0x0e 32  0  8  8  8  8 24  0  4 1 0x21TC      a  y  y  y     win,pb,pix
0x0f 32  0  8  8  8  8 24  0  8 1 0x21TC      a  y  y  y     win,pb,pix
0x10 32  0  8  8  8  8 24  0 16 1 0x21TC      a  y  y  y     win,pb,pix
0x11 32  0  8  8  8  8 24  8  2 1 0x21TC      a  y  y  y     win,pb,pix
0x12 32  0  8  8  8  8 24  8  4 1 0x21TC      a  y  y  y     win,pb,pix
0x13 32  0  8  8  8  8 24  8  8 1 0x21TC      a  y  y  y     win,pb,pix
0x14 32  0  8  8  8  8 24  8 16 1 0x21TC      a  y  y  y     win,pb,pix
0x15 24  0  8  8  8  0  0  0  0 0 0x21TC      y  y  y  y     win,pb,pix
0x16 24  0  8  8  8  0 16  0  0 0 0x21TC      y  y  y  y     win,pb,pix
0x17 24  0  8  8  8  0 24  0  0 0 0x21TC      y  y  y  y     win,pb,pix
0x18 24  0  8  8  8  0 24  8  0 0 0x21TC      y  y  y  y     win,pb,pix
0x19 24  0  8  8  8  0  0  0  2 1 0x21TC      y  y  y  y     win,pb,pix
0x1a 24  0  8  8  8  0  0  0  4 1 0x21TC      y  y  y  y     win,pb,pix
0x1b 24  0  8  8  8  0  0  0  8 1 0x21TC      y  y  y  y     win,pb,pix
0x1c 24  0  8  8  8  0  0  0 16 1 0x21TC      y  y  y  y     win,pb,pix
0x1d 24  0  8  8  8  0 16  0  2 1 0x21TC      y  y  y  y     win,pb,pix
0x1e 24  0  8  8  8  0 16  0  4 1 0x21TC      y  y  y  y     win,pb,pix
0x1f 24  0  8  8  8  0 16  0  8 1 0x21TC      y  y  y  y     win,pb,pix
0x20 24  0  8  8  8  0 16  0 16 1 0x21TC      y  y  y  y     win,pb,pix
0x21 24  0  8  8  8  0 24  0  2 1 0x21TC      y  y  y  y     win,pb,pix
0x22 24  0  8  8  8  0 24  0  4 1 0x21TC      y  y  y  y     win,pb,pix
0x23 24  0  8  8  8  0 24  0  8 1 0x21TC      y  y  y  y     win,pb,pix
0x24 24  0  8  8  8  0 24  0 16 1 0x21TC      y  y  y  y     win,pb,pix
0x25 24  0  8  8  8  0 24  8  2 1 0x21TC      y  y  y  y     win,pb,pix
0x26 24  0  8  8  8  0 24  8  4 1 0x21TC      y  y  y  y     win,pb,pix
0x27 24  0  8  8  8  0 24  8  8 1 0x21TC      y  y  y  y     win,pb,pix
0x28 24  0  8  8  8  0 24  8 16 1 0x21TC      y  y  y  y     win,pb,pix
0x29 32  0  8  8  8  8  0  0  0 0 0x22DC      a  y  y  y     win,pb,pix
0x2a 32  0  8  8  8  8 16  0  0 0 0x22DC      a  y  y  y     win,pb,pix
0x2b 32  0  8  8  8  8 24  0  0 0 0x22DC      a  y  y  y     win,pb,pix
0x2c 32  0  8  8  8  8 24  8  0 0 0x22DC      a  y  y  y     win,pb,pix
0x2d 32  0  8  8  8  8  0  0  2 1 0x22DC      a  y  y  y     win,pb,pix
0x2e 32  0  8  8  8  8  0  0  4 1 0x22DC      a  y  y  y     win,pb,pix
0x2f 32  0  8  8  8  8  0  0  8 1 0x22DC      a  y  y  y     win,pb,pix
0x30 32  0  8  8  8  8  0  0 16 1 0x22DC      a  y  y  y     win,pb,pix
0x31 32  0  8  8  8  8 16  0  2 1 0x22DC      a  y  y  y     win,pb,pix
0x32 32  0  8  8  8  8 16  0  4 1 0x22DC      a  y  y  y     win,pb,pix
0x33 32  0  8  8  8  8 16  0  8 1 0x22DC      a  y  y  y     win,pb,pix
0x34 32  0  8  8  8  8 16  0 16 1 0x22DC      a  y  y  y     win,pb,pix
0x35 32  0  8  8  8  8 24  0  2 1 0x22DC      a  y  y  y     win,pb,pix
0x36 32  0  8  8  8  8 24  0  4 1 0x22DC      a  y  y  y     win,pb,pix
0x37 32  0  8  8  8  8 24  0  8 1 0x22DC      a  y  y  y     win,pb,pix
0x38 32  0  8  8  8  8 24  0 16 1 0x22DC      a  y  y  y     win,pb,pix
0x39 32  0  8  8  8  8 24  8  2 1 0x22DC      a  y  y  y     win,pb,pix
0x3a 32  0  8  8  8  8 24  8  4 1 0x22DC      a  y  y  y     win,pb,pix
0x3b 32  0  8  8  8  8 24  8  8 1 0x22DC      a  y  y  y     win,pb,pix
0x3c 32  0  8  8  8  8 24  8 16 1 0x22DC      a  y  y  y     win,pb,pix
0x3d 24  0  8  8  8  0  0  0  0 0 0x22DC      y  y  y  y     win,pb,pix
0x3e 24  0  8  8  8  0 16  0  0 0 0x22DC      y  y  y  y     win,pb,pix
0x3f 24  0  8  8  8  0 24  0  0 0 0x22DC      y  y  y  y     win,pb,pix
0x40 24  0  8  8  8  0 24  8  0 0 0x22DC      y  y  y  y     win,pb,pix
0x41 24  0  8  8  8  0  0  0  2 1 0x22DC      y  y  y  y     win,pb,pix
0x42 24  0  8  8  8  0  0  0  4 1 0x22DC      y  y  y  y     win,pb,pix
0x43 24  0  8  8  8  0  0  0  8 1 0x22DC      y  y  y  y     win,pb,pix
0x44 24  0  8  8  8  0  0  0 16 1 0x22DC      y  y  y  y     win,pb,pix
0x45 24  0  8  8  8  0 16  0  2 1 0x22DC      y  y  y  y     win,pb,pix
0x46 24  0  8  8  8  0 16  0  4 1 0x22DC      y  y  y  y     win,pb,pix
0x47 24  0  8  8  8  0 16  0  8 1 0x22DC      y  y  y  y     win,pb,pix
0x48 24  0  8  8  8  0 16  0 16 1 0x22DC      y  y  y  y     win,pb,pix
0x49 24  0  8  8  8  0 24  0  2 1 0x22DC      y  y  y  y     win,pb,pix
0x4a 24  0  8  8  8  0 24  0  4 1 0x22DC      y  y  y  y     win,pb,pix
0x4b 24  0  8  8  8  0 24  0  8 1 0x22DC      y  y  y  y     win,pb,pix
0x4c 24  0  8  8  8  0 24  0 16 1 0x22DC      y  y  y  y     win,pb,pix
0x4d 24  0  8  8  8  0 24  8  2 1 0x22DC      y  y  y  y     win,pb,pix
0x4e 24  0  8  8  8  0 24  8  4 1 0x22DC      y  y  y  y     win,pb,pix
0x4f 24  0  8  8  8  0 24  8  8 1 0x22DC      y  y  y  y     win,pb,pix
0x50 24  0  8  8  8  0 24  8 16 1 0x22DC      y  y  y  y     win,pb,pix

Device platform:
eglinfo: eglInitialize failed
```

RedL0tus committed:
vulkan: bump REL

- Add vulkan-utility-libraries to PKGDEP.

RedL0tus committed:
intel-gpu-tools: update to 1.28

- The upstream has switched to meson, remove AUTOTOOLS_AFTER and ABSHADOW=0.
- Add patch for fixing overlay on non-x86 platforms.

RedL0tus committed:
glmark2: update to 2023.01

- The upstream has switched to meson, adjust options format accordingly.

RedL0tus committed:
mesa-demos: update to 9.0.0

RedL0tus committed:
peg: new, 0.1.18
